notifications.yaml

```diff
@@ -0,0 +1,275 @@
+        minLength: 3
+        maxLength: 100
+      body_long:
+        type: string
+        description: >-
+          A long version of the message, in plain text. The content of this
+          field will be delivered to channels that don't have any limit in terms
+          of content size (e.g. email, etc...). The long version of the message
+          is optional and should be provided only in cases when a longer message
+          (compared to the short version provided in body_short) gives more
+          information to the user. This field should not be a duplicate of
+          body_short.
+        minLength: 100
+    required:
+      - body_short
+  SubmittedMessage:
+    title: SubmittedMessage
+    type: object
+    properties:
+      dry_run:
+        type: boolean
+        description: >-
+          This parameter, when set to true, allows developers to test a request
+          without actually sending a message.
+
+
+          The default value is false.
+      time_to_live:
+        type: integer
+        description: >-
+          This parameter specifies for how long (in seconds) the system will try
+          to deliver the message to the channels configured by the user.
+        minimum: 3600
+        maximum: 31536000
+        default: 3600
+      content:
+        $ref: '#/definitions/MessageContent'
+    required:
+      - content
+  MessageStatus:
+    title: MessageStatus
+    type: object
+    properties:
+      status:
+        type: string
+        description: |
+          ACCEPTED: the message has been accepted for delivery
+
+          SENT: the message was delivered to the channel
+
+          FAILED: something wong happened, check the error code and message
+
+          TIMED_OUT:  in case a TTL was set
+        x-extensible-enum:
+          - ACCEPTED
+          - SENT
+          - TIMED_OUT
+          - FAILED
+      failed_at:
+        type: string
+        format: date-time
+      sent_at:
+        type: string
+        format: date-time
+      problem:
+        $ref: '#/definitions/ProblemJson'
+  AcceptedMessage:
+    title: AcceptedMessage
+    allOf:
+      - type: object
+        properties:
+          id:
+            type: string
+            format: uuid
+          accepted_at:
+            type: string
+            format: date-time
+          status:
+            type: object
+            default: {}
+            properties:
+              mail:
+                $ref: '#/definitions/MessageStatus'
+              sms:
+                $ref: '#/definitions/MessageStatus'
+              browser:
+                $ref: '#/definitions/MessageStatus'
+              app:
+                $ref: '#/definitions/MessageStatus'
+        required:
+          - id
+      - $ref: '#/definitions/SubmittedMessage'
+  PaginationResponse:
+    type: object
+    title: pagination
+    description: Pagination response parameters.
+    properties:
+      index:
```

Comment:
Also there should be an explanation of how pagination works on the description of the method that returns paginated results